imgreg/multi_spect_image_io.py

```diff
@@ -80,54 +80,43 @@ def load_bgr_image(path):
     return np.array(img)
 
 
+def _convert_to_degrees(value):
+    """
+    Convert the GPS coordinates stored in the EXIF to degress in float format.
+
+    :param value:
+    :type value: exifread.utils.Ratio
+    :rtype: float
+    """
+    d = float(value.values[0].num) / float(value.values[0].den)
```

Comment:
Can the denominator ever be 0?

Reply:
Those values come from GPSLatitude and GPSLongitude image metadata. Is it safe to assume those fields will never have denominators of 0?

Reply:
Not really, there are [valid fields](https://www.thoughtco.com/thmb/4Q7jZbyDMtYcoBFKkrBeaWGZByU=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/0-N-0-W-58d4164b5f9b58468375555d.jpg) with lat or lon equal or very close to 0.

Reply:
Addressed.